Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -567,7 +567,7 @@ void BrowserWindow::set_window_title_for_tab(Tab const& tab)
 
 void BrowserWindow::create_new_tab(URL url, bool activate)
 {
-    auto& new_tab = m_tab_widget->add_tab<Browser::Tab>("New tab", *this);
+    auto& new_tab = m_tab_widget->add_tab<Browser::Tab>("New tab"_short_string, *this);
```

Comment:
IIRC, any string longer than 3 bytes is too long for a short string, because we still support 32bit for wasm. I honestly don't know if that would apply here though.

Reply:
We already have short strings longer than 3 bytes elsewhere. Only stuff in the LibJS stack matters for 32-bit wasm, so this should be fine :)